specs/pvs/corla.pvs

```diff
@@ -132,14 +132,21 @@
 fafessl: THEORY
```

Comment:
Line 1989:
```
  ballots_to_audit_list_table: TYPE =
    sequence[[contest, sequence[ballot_manifest_info]]]
```
We should select ballots by county, not by contest. I'm not sure why we would store it this way.
The county selects from one list of ballots to audit, and audits all the contests on it.
When multiple contests are being audited, we wouldn't want disjoint sets of selections.
When a county only appears on a subset of contests, at this point we just present them all
and work until they audit enough of them.  When, in some future version, we want to give them a filtered list by contest, we may want to filter by multiple (still active) contests, not just one.
Cross-county contests are audited via a yet-to-be solidified stratified audit procedure.